tox.ini

```diff
@@ -1,17 +1,24 @@
 [tox]
-envlist = tests, flake8
+envlist = py{27,35}, marionette, flake8
```

Comment:
Thanks for the comment. But I removed it on purpose (see my comments and commits). I will see what could  be done with the `matrix` field.